File: Bit Manipulation/BitManipulation1.java
import java.util.Scanner;

class BitManipulation1{
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        System.out.print("Enter a number to Check Even or Odd : ");
        int n=sc.nextInt();
        if((n & 1)==1){
            System.out.println(n+" is Odd Number");
        }else{
            System.out.println(n+" is Even Number");
        }
        sc.close();
    }
}

File: Bit Manipulation/BitManipulation1.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
    int n ;
    cout<<"Enter a number : "<<endl;
    cin>>n;
    if ((n & 1) == 1)
    {
        cout<<n<<" is Odd Number";
    }
    else
    {
        cout<<n<< " is Even Number";
    }

    return 0;
}